app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,161 @@ function Page() {
-                    display: grid;
+    display: grid;
 
-                    padding: var(--_gap);
-                    grid-gap: var(--_gap);
-                    grid-template-columns: repeat(4, 1fr);
-                    grid-template-rows: auto;
-                  }
-                `}
-                </code></pre>
-                </div>
+    padding: var(--_gap);
+    grid-gap: var(--_gap);
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+  }
+`}
+                </code></pre></div>
               </li>
             </ul>
 
-            <h2>Layout</h2>
-            <p>TBD ...</p>
+            <h2><code>Layout Container</code> (Base)</h2>
+            <p>The stanard <code>{`<LayoutContainer>`}</code> wraps content containers to provide
+            some common container within the layout container.</p>
+            <LayoutContainer style={{'--layout-container-background': 'var(--slate-200)'}}>
+              <ContentContainer>
+                <div><pre><code>
+                  {`<LayoutContainer style={{'--layout-container-background': 'var(--slate-100)'}}>
+  <ContentContainer> ... </ContentContainer>
+</LayoutContainer>
+`}
+                </code></pre></div>
+              </ContentContainer>
+            </LayoutContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-container-padding</code></dt>
+              <dd>
+                Set the container padding, defaults to: <code>0</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>Content Container</code> (Base)</h2>
+            <ContentContainer style={{'border': '1px solid var(--slate-600)'}}>
+              <p>This conteint is wrapped in the following container.</p>
+              <p><strong>Note</strong> that the style property is not required
+                to use the container. It's only here to show that you can use it
+                this way, and to show the outline here in the styleguide.</p>
+              <div><pre><code>{`<ContentContainer style={{'border': '1px solid var(--purple-600)'}}>
+...
+</ContentContainer>`}
+              </code></pre></div>
+            </ContentContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-content-padding</code></dt>
+              <dd>
+                Set the content container padding, defaults to: <code>var(--brand-space2)</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>ToolbarContainer</code> (LayoutContainer)</h2>
+            <p><code>LayoutContainer > ToolbarContainer</code></p>
```

Comment:
L1129: Can you change this line "<p><code>LayoutContainer > ToolbarContainer</code></p>" to use "&gt;" because it is throwing a linting error. So final one should look like "<p><code>LayoutContainer &gt; ToolbarContainer</code></p>"